pkg/controller/certificates/rootcacertpublisher/publisher.go

```diff
@@ -36,23 +36,18 @@ import (
 	"k8s.io/kubernetes/pkg/util/metrics"
 )
 
-// RootCACertCofigMapName is name of the configmap which stores certificates to access api-server
+// RootCACertCofigMapName is name of the configmap which stores certificates to
+// access api-server
 const RootCACertCofigMapName = "kube-root-ca.crt"
```

Comment:
RootCACertCofigMapName : typo RootCACertConfigMapName

Reply:
This is being fixed in https://github.com/kubernetes/kubernetes/pull/70833